src/main/resources/openapi/sqs/Notifications-sqs-messaging.yaml

```diff
@@ -723,10 +735,13 @@ components:
         hasRestrictedDocumentsList:
```

Comment:
Worth considering renaming this object now that it's being repurposed

If that's too much work and out of scope, we should be making a tech debt entry/ticket and at least changing the object description (won't require changes in the non-OE services)

Reply:
Spoke to Will and he said to make a separate ticket in the epic that can be done when we have abit more time (maybe next week) https://dluhcdigital.atlassian.net/browse/EIP1-9093